### ISSUE TITLE
Removing a pvrdma device fails

### ISSUE THREAD

**Issue by earlruby** [vmware/vhpc-toolkit#4]
**Describe the bug**
Removing a pvrdma device fails.

**To Reproduce**
```
vhpc_toolkit> pvrdma --vm dilemma.example.com --remove --pvrdma_port_group 'VM 100GbE Network'
2020-04-13 14:41:32,182 GetObjects [INFO] Found vm dilemma.example.com
Traceback (most recent call last):
  File "./vhpc_toolkit", line 12, in <module>
    Cmd().cmdloop()
  File "/Users/eruby/.pyenv/versions/3.7.4/lib/python3.7/cmd.py", line 138, in cmdloop
    stop = self.onecmd(line)
  File "/Users/eruby/.pyenv/versions/3.7.4/lib/python3.7/cmd.py", line 217, in onecmd
    return func(arg)
  File "/Users/eruby/Projects/vhpc-toolkit/venv/lib/python3.7/site-packages/vhpc_toolkit-0.1.0-py3.7.egg/vhpc_toolkit/command.py", line 185, in do_pvrdma
    ops.pvrdma_cli()
  File "/Users/eruby/Projects/vhpc-toolkit/venv/lib/python3.7/site-packages/vhpc_toolkit-0.1.0-py3.7.egg/vhpc_toolkit/operations.py", line 1481, in pvrdma_cli
    tasks.append(self._get_remove_pvrdma_tasks(vm_cfg))
  File "/Users/eruby/Projects/vhpc-toolkit/venv/lib/python3.7/site-packages/vhpc_toolkit-0.1.0-py3.7.egg/vhpc_toolkit/operations.py", line 1559, in _get_remove_pvrdma_tasks
    network_name=pg, device_type=vim.VirtualVmxnet3Vrdma
  File "/Users/eruby/Projects/vhpc-toolkit/venv/lib/python3.7/site-packages/vhpc_toolkit-0.1.0-py3.7.egg/vhpc_toolkit/get_objs.py", line 774, in network_obj
    if isinstance(dev, device_type) and dev.backing.deviceName == network_name:
AttributeError: 'vim.vm.device.VirtualEthernetCard.DistributedVirtu' object has no attribute 'deviceName'
```

**Expected behavior**
Expected the pvrdma device to be removed.

**Additional context**
The list of objects returned by `get_objs.py: network_obj` includes `vim.vm.device.VirtualEthernetCard.DistributedVirtualPortBackingInfo`, which does not have a `deviceName` attribute.

I tried adding an exception handler for the `AttributeError`, but there are no objects returned with the 'VM 100GbE Network' port name group as the `deviceName`, so even after handling the error there are no network objects available to remove.

**Comment by Michael870** [comment]
This is likely to be an issue brought by committing new changes without sufficient regression test. I am able to reproduce your error, and by reverting back to previous version before the last few commits, I am able to execute the operation successfully. The easiest fix is to revert back to commit fc4eee48bf86e175ebf5db06c3fa1bb58650e451

**Comment by nazhang8** [comment]
this issue has been fixed in master.
